examples/tools/generate.ipynb

```diff
@@ -0,0 +1,514 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kinematics.create_all_subsystems()\n",
+    "phsp_set = kinematics.convert(phsp_sample)\n",
+    "phsp_set_importance = kinematics.convert(phsp_sample_importance)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "We can then use :mod:`matplotlib.pyplot` to create the Dalitz plot:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import LogNorm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transition_info = kinematics.get_particle_state_transition_kinematics_info()\n",
+    "id_to_name = transition_info.get_final_state_id_to_name_mapping()\n",
+    "\n",
+    "\n",
+    "def replace_ids(title):\n",
+    "    \"\"\"Replace all IDs in a string by particle names.\"\"\"\n",
+    "    for id, name in id_to_name.items():\n",
+    "        title = title.replace(str(id), name)\n",
+    "    return title\n",
+    "\n",
+    "\n",
+    "def dalitz_plot(data_set, mass_x, mass_y, bins=50, **kwargs):\n",
+    "    \"\"\"Helper function to create a Dalitz plot with useful axis titles.\"\"\"\n",
+    "    plt.hist2d(\n",
+    "        data_set.data[mass_x],\n",
+    "        data_set.data[mass_y],\n",
+    "        weights=data_set.weights,\n",
+    "        bins=bins,\n",
+    "        **kwargs)\n",
+    "    plt.xlabel(replace_ids(mass_x))\n",
+    "    plt.ylabel(replace_ids(mass_y))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dalitz_plot(phsp_set_importance, 'mSq_(3,4)', 'mSq_(2,4)',\n",
+    "            bins=80, norm=LogNorm())\n",
+    "plt.gca().set_title('importance-sampled phase space');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note how this compares to the evenly distributed phase-space sample and the\n",
+    "intensity-based sample:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dalitz_plot(phsp_set, 'mSq_(3,4)', 'mSq_(2,4)', bins=80, norm=LogNorm())\n",
+    "plt.gca().set_title('evenly distributed phase space');"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pwa.Logging('error')\n",
+    "data_sample = pwa.generate(\n",
+    "    2000, kinematics, generator, intensity, random_generator)\n",
+    "data_set = kinematics.convert(data_sample)\n",
+    "dalitz_plot(data_set, 'mSq_(3,4)', 'mSq_(2,4)', bins=80, norm=LogNorm())\n",
+    "plt.gca().set_title('intensity-based sample');"
+   ]
+  }
```

Comment:
3rd data sample. I think here it fits best though. So I would leave this one

Reply:
Yeah probably one of the other `data_sample` can be reused here, it's just to plot the distributions

Reply:
https://github.com/ComPWA/pycompwa/pull/79/commits/4c318d52a08ba92ffdccf9915afabe87d5328080